content/support-matrix/delfin/add-driver-support/_index.md

```diff
@@ -0,0 +1,434 @@
+    ---
+    
+    Support matrix data for Dell EMC Models can be found here
+    
+    ```
+
+{{% notice note %}}
+Please ensure you copy the entire frontmatter including both the `---`
+{{% /notice %}}
+    
+{{% notice note %}}
+Step 3, 4 and 5 are not required if there is already vendor folder name exists in the documentation repository
+{{% /notice %}}
+
+6) Under vendor folder name, create a new folder with `model` name. For example `VNX`.
+7) Create a new file `_index.md`. 
+8) Add the frontmatter as follows. Change the content as required.
+    ```
+    ---
+    title: VNX
+    menuTitle: VNX 
+    description: "VNX Support Matrix"
+    weight: 20
+    disableToc: false
+    tags: ["support matrix", "delfin", "drivers", "dell emc"]
+    ---
+    ```
+    
+{{% notice note %}}
+Please ensure you copy the entire frontmatter including both the `---`
+{{% /notice %}}
+
+9) To add all the support matrix tables content, copy lines from 57 to 432 from a file content/support-matrix/delfin/add-driver-support/_index.md 
+10) Fill appropriate driver data to support matrix tables. 
+11) Please find below sample driver support matrix data tables with sample data for reference to fill specific driver support matrix data.
+
+### Model and management software versions supported
+
+<table>
+   <thead>
+      <tr>
+         <th>Models</th>
+         <th>Management Software Versions</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td>Sample model name</td>
+         <td>Management software version</td>
+      </tr>
+      <tr>
+         <td></td>
+         <td></td>
+      </tr>
+   </tbody>
+</table>
+
+### Access information to register storage device
+
+<table>
+   <thead>
+      <tr>
+         <th>Access Type</th>
+         <th>Attributes</th>
+         <th>Additional information</th>
+      </tr>
+   </thead>
+   <tbody>
+      <tr>
+         <td rowspan="4">REST</td>
+         <td>Host</td>
+         <td>Management software version host name</td>
+      </tr>
+      <tr>
+         <td>Port</td>
+         <td>Management software version port name</td>
+      </tr>
+      <tr>
+         <td>Username</td>
+         <td>Management software version Username</td>
+      </tr>
+      <tr>
+         <td>Password</td>
+         <td>Management software version Password</td>
+      </tr>
+      <tr>
+         <td rowspan="4">SSH</td>
+         <td>Host</td>
+         <td>Management software version host name</td>
+      </tr>
+      <tr>
+         <td>Port</td>
+         <td>Management software version port name</td>
+      </tr>
+      <tr>
+         <td>Username</td>
+         <td>Management software version Username</td>
+      </tr>
+      <tr>
+         <td>Password</td>
+         <td>Management software version Password</td>
+      </tr>
+      <tr>
+         <td></td>
+         <td></td>
+      </tr>
+   </tbody>
+</table>
+
+### Alert management information
+<table>
+   <tbody>
+      <tr>
+          <th>SNMP trap processing</th> 
```

Comment:
Why the `<th>` inside a row and why all the rows are spanned together?
![image](https://user-images.githubusercontent.com/19162717/109104334-cd93bd80-7751-11eb-8f9d-75f1b0ef82d4.png)

Reply:
It's default table style rendering has issue

Reply:
Do we need a row header? We can just have a regular table as we have for the other information.

Reply:
@Shruthi-1MN 
Instead of a `<th>` can you try using a regular `<td>` and put the content inside `<strong>This is bold text</strong>` For ex: <strong>This is bold text</strong>.  
```html
<table>
<tr>
<td><strong>Query Backend Alert</strong></td>
<td>Supported</td>
</tr>
</table>
```
<table>
<tr>
<td><strong>Query Backend Alert</strong></td>
<td>Supported</td>
</tr>
</table>